x-pack/plugins/security_solution/public/management/pages/event_filters/view/components/event_filter_delete_modal.test.tsx

```diff
@@ -39,6 +40,12 @@ describe('When event filters delete modal is shown', () => {
 
   const getCurrentState = () => store.getState().management.eventFilters;
 
+  const partialEventFilter: Partial<ExceptionListItemSchema> = {
```

Comment:
can you turn this into a function that return the object?
We should try to avoid mutable state between tests in order to ensure one test does not impact the next one (ex. through an accidental mutation of the object's values

Reply:
You could also just explicitly set the variable inside of the `beforeEach()` instead of turning it into a function

Reply:
you can also use `getExceptionListItemSchemaMock` which accepts overwrites like this:

`getExceptionListItemSchemaMock({ tags: []});`